.circleci/config.yml

```diff
@@ -1,71 +1,56 @@
+            chmod +x bazel-0.24.0-installer-linux-x86_64.sh
+            sudo ./bazel-0.24.0-installer-linux-x86_64.sh
+            rm ./bazel-0.24.0-installer-linux-x86_64.sh
+            bazel info release
+  restore-caches:
+    steps:
+      - restore_cache:
+          keys:
+            - "android_sdk"
+      - restore_cache:
+          keys:
+            - "v8-bazel_cache"
 
+  save-caches:
+    steps:
+      - save_cache:
+          key: "android_sdk"
+          paths:
+            - "/home/circleci/android_sdk"
 
-# These jobs will run in parallel, and report separate statuses to GitHub PRs
+      - save_cache:
+          key: "v8-bazel_cache"
+          paths:
+            - "/home/circleci/.cache/bazel/"
 jobs:
   lint:
-    <<: *job_defaults
+    working_directory: ~/ng/
+    machine:
+      enabled: true
+      image: ubuntu-1604:201903-01
     steps:
-      - checkout:
-          <<: *post_checkout
-      - restore_cache:
-          key: "v7-bazel_cache"
-      - run: './tools/pre-commit.sh'
-      # Fixes Debian upstream repos: https://discuss.circleci.com/t/failed-to-fetch-jessie-updates/29246
-      - run: |
-          sudo rm /etc/apt/sources.list
-          echo "deb http://archive.debian.org/debian/ jessie-backports main" | sudo tee -a /etc/apt/sources.list
-          echo "deb-src http://archive.debian.org/debian/ jessie-backports main" | sudo tee -a /etc/apt/sources.list
-          echo "Acquire::Check-Valid-Until false;" | sudo tee -a /etc/apt/apt.conf.d/10-nocheckvalid
-          echo 'Package: *\nPin: origin "archive.debian.org"\nPin-Priority: 500' | sudo tee -a /etc/apt/preferences.d/10-archive-pin
-          sudo apt-get update
-      - run: sudo apt -y install yapf3 python3-pkg-resources libstdc++6
-      - run: sudo ln -s /usr/bin/yapf3 /usr/bin/yapf
+      - checkout
+      - install-bazel
       - run: './tools/deps/check_dependencies.sh'
       - run: './tools/fix_formatting.sh'
 
   build:
-    <<: *job_defaults
+    working_directory: ~/ng/
+    machine:
+      enabled: true
+      image: ubuntu-1604:201903-01
     steps:
-      - checkout:
-          <<: *post_checkout
-
-      - restore_cache:
-          key: "android_sdk"
-
-      - restore_cache:
-          key: "v7-bazel_cache"
-
-      # Fixes Debian upstream repos: https://discuss.circleci.com/t/failed-to-fetch-jessie-updates/29246
-      - run: |
-          sudo rm /etc/apt/sources.list
-          echo "deb http://archive.debian.org/debian/ jessie-backports main" | sudo tee -a /etc/apt/sources.list
-          echo "deb-src http://archive.debian.org/debian/ jessie-backports main" | sudo tee -a /etc/apt/sources.list
-          echo "Acquire::Check-Valid-Until false;" | sudo tee -a /etc/apt/apt.conf.d/10-nocheckvalid
-          echo 'Package: *\nPin: origin "archive.debian.org"\nPin-Priority: 500' | sudo tee -a /etc/apt/preferences.d/10-archive-pin
```

Comment:
We don't need `yapf3`? How do we format python code?

Reply:
@oferb we don't have Python code in `@startup_os`/`@hasadna`/`@devenv` — at least as far as I've checked — so I decided to simplify config to not install `yapf`

Reply:
We have some python code on the way, and I wouldn't want to lose support for it.
Could you reinstate it?

Reply:
@oferb fixed in https://github.com/google/startup-os/commit/8bbd11a136c77a768a922fab21b6a9f544cf6ed1